### PR TITLE
Add Python Tour of Restate

### PR DESCRIPTION
**Pull request by gvdongen**

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
The durable animation here still uses typescript code

**Comment by gvdongen** [comment]
> The durable animation here still uses typescript code

I have it on my planning to make this animation work for smaller screens. But I don't think I will replicate it for each language. I think the gist is clear no matter the exact language used

File: .github/workflows/test-build.yml [hunk 1]
```diff
@@ -40,6 +40,24 @@ jobs:
...
+        run: |
+          cd code_snippets/python
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -r requirements.txt
+          pip install mypy
+          python3 -m mypy .
+          deactivate
+
       # Setup Java
```
Comment (slinkydeveloper) [inline .github/workflows/test-build.yml:61]: It seems to me this is not running?!

Comment (gvdongen) [reply .github/workflows/test-build.yml:61]: Solved it in another PR

**Pull request merged**